From ac2f7d9287c7aa2090ad6a49a29e55252a65a5ea Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Mon, 18 Oct 2021 12:15:26 +0200
Subject: [PATCH 304/478] mfd: rk808: Drop reset handlers for rk805, rk808 and
 rk818

They don't work. The PMIC just powers off and resets its internal state.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 drivers/mfd/rk808.c | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/drivers/mfd/rk808.c b/drivers/mfd/rk808.c
index b608264f52d2..2405b32b8132 100644
--- a/drivers/mfd/rk808.c
+++ b/drivers/mfd/rk808.c
@@ -569,18 +569,6 @@ static int rk808_restart_notify(struct notifier_block *this, unsigned long mode,
 	struct rk808 *rk808 = i2c_get_clientdata(rk808_i2c_client);
 
 	switch (rk808->variant) {
-	case RK805_ID:
-		reg = RK805_DEV_CTRL_REG;
-		bit = DEV_OFF_RST;
-		break;
-	case RK808_ID:
-		reg = RK808_DEVCTRL_REG,
-		bit = DEV_OFF;
-		break;
-	case RK818_ID:
-		reg = RK818_DEVCTRL_REG;
-		bit = DEV_OFF_RST;
-		break;
 	case RK817_ID:
 		reg = RK817_SYS_CFG(3);
 		bit = DEV_RST;
-- 
2.35.3

